ts/nni_manager/rest_server/restValidationSchemas.ts

```diff
@@ -226,6 +226,7 @@ export namespace ValidationSchemas {
             trainingServicePlatform: joi.string(),
             searchSpace: joi.string().required(),
             maxExecDuration: joi.number().min(0).required(),
+            maxTrialDuration: joi.number().min(0).required(),
```

Comment:
where is "STARTEXPERIMENT" used?

Reply:
It is validation for V1 config to start experiment, validation not enabled now, but we had added the field, and should add it here.